packages/core/src/lib/Table/Table.styles.ts

```diff
@@ -93,6 +95,9 @@ const TableGrid = styled.div.attrs((props: any) => ({
     & > div.table_header {
         font-weight: 600;
         padding: 12px 0;
+        color: ${(props) => props.headerTextColor ?? 'white'};
```

Comment:
can you use `color.white` please
color is import from @web3uikit/styles package